advanced-vector-search/README.md

```diff
@@ -46,19 +56,20 @@ Before moving forward, we highly suggest completing/reviewing our lovely [Jina 1
 
 ## Requirements
 
-Be sure to create an environment with Python 3.7 or higher installed.
+Be sure to create an environment with Python 3.7 or higher installed, then you also need to have the requirements of this example,
+which is `jina` and `click`.
+
+In order to run the example with the different indexers, make sure to pull the docker images from the [Jina Hub repository] (https://hub.docker.com/u/jinahub)
 
-Next, open a terminal and run:
+## Prepare the data
 
-```Bash
-wget https://raw.githubusercontent.com/jina-ai/examples/master/faiss-search/requirements.txt
+In order to get the data needed to run the example, we have prepared a small script that will download the required fails.
 
-pip install -r requirements.txt
+```bash
+./get_siftsmall.sh
 ```
 
-Additionally, since FAISS introduces many dependencies, the [Jina hub image](https://github.com/jina-ai/jina-hub/tree/master/indexers/vector/FaissIndexer) is used for simplicity.
-
-## Prepare the data
+Moreover, 
```

Comment:
It has a weird format, maybe a new line was added?

Reply:
well there is a line between Moreover and Faiss, maybe we can refactor later